src/mirage/mirage.ml

```diff
@@ -1,3 +1,5 @@
+[@@@warning "-32"]
```

Comment:
What exactly is this suppressing? Can it be made more local?

Reply:
I need to lint a bit the whole API to see what is really useless and what is really needed but not yet used.